tests/services/test_availability_service.py

```
@@ -255,3 +255,79 @@ async def test_full_flow_with_mock_data(self, service, mock_crawler, mock_pricin
 
         # 3. DB Mock이 호출되었는지
         mock_db.assert_called_once()
+
+    @pytest.mark.asyncio
+    async def test_aggregate_min_price_by_available_status(self, service, mock_crawler, mock_pricing_service):
```

Ruff (0.15.2)

[warning] Unused method argument: `mock_pricing_service`

(ARG002)